exasol_advanced_analytics_framework/event_handler/event_handler_result.py

```diff
@@ -11,7 +11,16 @@ class EventHandlerReturnQuery:
 
 
 @dataclass()
-class EventHandlerResult:
-    return_query: Optional[EventHandlerReturnQuery]
+class EventHandlerResultBase:
     status: str
```

Comment:
I think, we don't need the status attribute anymore. We can derive it from the Class Type of the result object. This is also safer, because the event loop expects specific status values